lecture/forms.py

```
@@ -1,17 +1,26 @@
+        widgets = {
+            'course': forms.Select(attrs={'class': 'form-select'})  # Fix select field styling
+        }
 
 class LecturePDFForm(forms.ModelForm):
     class Meta:
         model = LecturePDF
         fields = ['pdf_file', 'lecture']
+        widgets = {
+            'lecture': forms.Select(attrs={'class': 'form-select'}),  # Fix select field styling
+            'pdf_file': forms.FileInput(attrs={'class': 'form-control'})  # File input styling
+        }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)